MAINTAINERS.md

```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+The oneDPL project provides algorithms that are implemented using parallel patterns and bricks implemented
+for the various backends supported to allow oneDPL to run on CPU, GPU, and FPGA platforms. The API areas
+which a developer is a lead maintainer are listed below.
+
+
+| Name                  | Github ID           | API Area |
+| --------------------- | ------------------- | -------- |
+| Ruslan Arutyunyan     | @rarutyun           | PSTL Offload |
+| Konstantin Boyarinov  | @kboyarinov         | PSTL Offload |
```

Comment:
Since PSTL Offload is a compiler feature, not oneDPL one, should it still be present in this list?

Reply:
I'm not sure what the timeline is and how much of the code will remain here in oneDPL's repo, but I think as long as there is code living in the repo related to PSTL Offload, its a reasonable thing to have in this list.

Reply:
I agree with Dan. My plan was to update this table when the code migrated.